pkg/util/helm/interface.go

```diff
@@ -0,0 +1,10 @@
+package helm
+
+import (
+	helmclient "github.com/mittwald/go-helm-client"
+)
+
+type Client interface {
```

Comment:
What is the purpose of this interface? Adding some comments will be great. By the way, I can't find where this interface is used.

Reply:
@hxcGit This interface is used to generate mockclient by mockgen